APE_coordinated_pkg.rst

```diff
@@ -0,0 +1,235 @@
+An Updated Model for the Affiliated Package Ecosystem
+-----------------------------------------------------
+
+author: Erik Tollerud, 2017 Astropy Coordination Meeting Attendees
+
+date-created: 2018 February 22
+
+date-last-revised: 2018 February 22
+
+type: Process
+
+status: Discussion
+
+
+Abstract
+--------
+
+A revision of the concept of "affiliated packages" is proposed.  The idea of
+affiliated packages has continued mostly unaltered since the original Vision
+for Astropy.  However, time has revealed some potential improvements in how the
+ecosystem is structured and the review process for affiliated packages.  Hence,
+this APE proposes to make three changes:
+
+* This APE creates the concept of "coordinated" packages - affiliated-like packages
+  that are more closely overseen by the Astropy Project. While apparently
+  significant, this change is mainly a recognition of de facto status that some
+  affiliated packages already have.
+* The APE changes the review process to a more explicit criteria-based review
+  process - again, this is primarily simply formalizing a process that was
+  already in place in the coordination committee.
+* It lays out guidelines for the eventual goal of converting the
+  affiliated package review process into a more "peer-review" style, where
+  other members of the community can contribute to the review.
+
+
+Detailed description
+--------------------
+
+Motivation
+==========
+
+From its inception, Astropy has included an ecosystem of astronomy packages
+beyond the ``astropy`` core package (see the
+`original vision <http://docs.astropy.org/en/stable/development/vision.html>`_).
+The process by which these "affiliated" packages are reviewed and accepted has
+already changed a few times since the project's inception as the ecosystem has
+grown. However, the current state has several significant limitations.
+
+First, the affiliated packages include a mix of two different kinds of packages.
+Some packages have been developed as a fairly integrated effort alongside the
+core package.  That is, feature planning took into account that these
```

Comment:
Here and elsewhere, remove double spaces after periods 😛

Reply:
:trollface: - but I'll do it! :wink: